packages/run-it/package.json

```diff
@@ -60,8 +59,7 @@
     "react-dom": "^16.13.1",
     "react-google-charts": "^3.0.15",
     "react-is": "^16.13.1",
-    "styled-components": "^4.4.1",
-    "styled-system": "^5.1.5"
```

Comment:
`styled-system` dependency no longer needed (was used for `Layout` components previously)